tools/spocc/spocc_occ.xml

```diff
@@ -0,0 +1,93 @@
+<tool id="spocc_occ" name="Get species occurrences data" version="0.9.0">
+    <description>from GBIF, ALA, iNAT and others</description>
+    <requirements>
+        <requirement type="package" version="0.9.0">r-spocc</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript '$__tool_directory__/spocc_occ.R' 
+            '$sname' 
+            '$from' 
+            $limit; 
+
+        #if $from=='gbif':
+            sed -z -i -e 's/\n  \"/ /g' -e 's/\n}/ /g' output.tab
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="sname" type="text" label="Scientific name" value="Genus species"/>
+        <param name="from" type="select" multiple="true" label="Data source to get data from" help="Any combination of gbif, bison, inat, ebird, antweb, ala, idigbio, obis, ecoengine and/or vertnet" value="gbif,bison,inat">
+            <option value="gbif">gbif</option>
```

Comment:
Do you want to add some more info to these data sources?

Reply:
Yes good point, I should give the full names of databases here and add links in the help section